README.md

```diff
@@ -60,6 +60,13 @@ with the `read:packages` scope.
 @trussworks:registry=https://npm.pkg.github.com
 ```
 
+or
```

Comment:
I think this prerelease section should go into For Trussels docs instead. Fine with merging this as is though and cleaning up because there are some other small changes to the README I was planning to make for Hacktober fest.